api/generate_models.ts

```diff
@@ -68,3 +69,8 @@ generateModelsFromApi(env, "public_api_v1", "lib/api").then(
   () => console.log("done"),
   err => console.log(`Error: ${err}`)
 );
+
+generateModelsFromApi(env, "admin_api", "lib/api").then(
```

Comment:
it looks like this method responsibility is not just generating models anymore, it's worth changing its name to something else, like `exportApi` or `generateApi`

Reply:
ok